### ISSUE TITLE
Custom Texts for C type bsod?

### ISSUE THREAD

**Issue by RTFTR** [blueskychan-dev/FuckedUP#2]
I noticed that the c000021a bsod on your example(WinXP SP4) had (null) text, which seems to be able to be edited.

I wonder if you can make some codes to make it customizable? I'll look forward to your response.

**Comment by blueskychan-dev** [comment]
Technically, it possible but 

1. You need administrator
2. I need to modify WinNT side

But i still research about this, so if anything going great, i will push the feature to main branch :p

**Comment by RTFTR** [comment]
I remember there was a video of russian coder showing himself making bsod in C++. It seems to be gone but i believe you can work this out

**Comment by RTFTR** [comment]
Okay so, after some tries without knowledge of C++, the 4th argument of NtRaiseHardError is responsible for the (null) text, and UAC permission wasn't required in order to set the custom message. I believe you can do it easily.

**Comment by blueskychan-dev** [comment]
> Okay so, after some tries without knowledge of C++, the 4th argument of NtRaiseHardError is responsible for the (null) text, and UAC permission wasn't required in order to set the custom message. I believe you can do it easily.

Do you have any example of code or video, so i can try.
if anything going great, i will try make new version of fuckedup.

I wait for your reply :D

**Comment by RTFTR** [comment]
> > Okay so, after some tries without knowledge of C++, the 4th argument of NtRaiseHardError is responsible for the (null) text, and UAC permission wasn't required in order to set the custom message. I believe you can do it easily.
> 
> Do you have any example of code or video, so i can try.
> if anything going great, i will try make new version of fuckedup.
> 
> I wait for your reply :D

https://github.com/RTFTR/BSOD-Custom-Parameter

Here's my repo!

**Comment by blueskychan-dev** [comment]
> > > Okay so, after some tries without knowledge of C++, the 4th argument of NtRaiseHardError is responsible for the (null) text, and UAC permission wasn't required in order to set the custom message. I believe you can do it easily.
> > 
> > 
> > Do you have any example of code or video, so i can try.
> > if anything going great, i will try make new version of fuckedup.
> > I wait for your reply :D
> 
> https://github.com/RTFTR/BSOD-Custom-Parameter
> 
> Here's my repo!

Great, Thanks for your repo!
I will try this, Thanks again!

**Comment by blueskychan-dev** [comment]
> > > Okay so, after some tries without knowledge of C++, the 4th argument of NtRaiseHardError is responsible for the (null) text, and UAC permission wasn't required in order to set the custom message. I believe you can do it easily.
> > 
> > 
> > Do you have any example of code or video, so i can try.
> > if anything going great, i will try make new version of fuckedup.
> > I wait for your reply :D
> 
> https://github.com/RTFTR/BSOD-Custom-Parameter
> 
> Here's my repo!

Thanks for your help!
Sadly, i need Windows 7 to test this since i already try with windows 10 (my laptop) but i got only error code...
I gonna try with my VM

**Comment by RTFTR** [comment]
> > > > Okay so, after some tries without knowledge of C++, the 4th argument of NtRaiseHardError is responsible for the (null) text, and UAC permission wasn't required in order to set the custom message. I believe you can do it easily.
> > > 
> > > 
> > > Do you have any example of code or video, so i can try.
> > > if anything going great, i will try make new version of fuckedup.
> > > I wait for your reply :D
> > 
> > https://github.com/RTFTR/BSOD-Custom-Parameter
> > 
> > Here's my repo!
> 
> Thanks for your help!
> Sadly, i need Windows 7 to test this since i already try with windows 10 (my laptop) but i got only error code...
> I gonna try with my VM

Yeah, I put some additional info in the README.md of my repo, lol
